k8s/aat/common/sscs/sscs-tya-notif.yml

```diff
@@ -22,7 +22,7 @@ spec:
       cpuLimits: '1500m'
       useInterpodAntiAffinity: true
       applicationInsightsInstrumentKey: "e2300a5f-a16f-4657-a63b-41913805661e"
-      image: hmctspublic.azurecr.io/sscs/tya-notif:prod-e07d0898
```

Comment:
not sure why u want to do this ? flux will manage tags based on https://github.com/hmcts/cnp-flux-config/pull/753/files#diff-4ab970973ca77db6a0ac8017993800d0L9

Reply:
Thanks @adusumillipraveen - @jackmaloney mentioned that we might need to set it to "latest" the first time around, but if Flux will manage it in any case, that's great. Was just hoping it might be the reason we can't get it to deploy :(

Reply:
Shouldn't be the case.

Reply:
you can commit latest in the first commit, and then flux manages it for you from then on